tools/gromacs/energy.xml

```diff
@@ -22,55 +22,57 @@
-            <option value="1">Angle</option>
-            <option value="2">Proper dihedrals</option>
-            <option value="3">Ryckaert-Bellemans dihedrals</option>
-            <option value="4">Lennard-Jones (1-4 interactions)</option>
-            <option value="5">Coulomb (1-4 interactions)</option>
-            <option value="6">Lennard-Jones energies (short range)</option>
-            <option value="7">Dispersion correction</option>
-            <option value="8">Coulomb (short range)</option>
-            <option value="9">Reciprocal Coulomb energy</option>
-            <option value="10" selected="True">Potential</option>
-            <option value="11">Kinetic energy</option>
-            <option value="12">Total energy</option>
-            <option value="13">Temperature</option>
-            <option value="14">Pressure DC</option>
-            <option value="15">Pressure</option>
-            <option value="16">Constr. RMSD</option>
-            <option value="17">Box-X</option>
-            <option value="18">Box-Y</option>
-            <option value="19">Box-Z</option>
-            <option value="20">Volume</option>
-            <option value="21">Density</option>
-            <option value="22">pV</option>
-            <option value="23">Enthalpy</option>
-            <option value="24">Virial(XX)</option>
-            <option value="25">Virial(XY)</option>
-            <option value="26">Virial(XZ)</option>
-            <option value="27">Virial(YX)</option>
-            <option value="28">Virial(YY)</option>
-            <option value="29">Virial(YZ)</option>
-            <option value="30">Virial(ZX)</option>
-            <option value="31">Virial(ZY)</option>
-            <option value="32">Virial(ZZ)</option>
-            <option value="33">Pressure(XX)</option>
-            <option value="34">Pressure(XY)</option>
-            <option value="35">Pressure(XZ)</option>
-            <option value="36">Pressure(YX)</option>
-            <option value="37">Pressure(YY)</option>
-            <option value="38">Pressure(YZ)</option>
-            <option value="39">Pressure(ZX)</option>
-            <option value="40">Pressure(ZY)</option>
-            <option value="41">Pressure(ZZ)</option>
-            <option value="42">Surface tension</option>
-            <option value="43">Box-Vel-XX</option>
-            <option value="44">Box-Vel-YY</option>
-            <option value="45">Box-Vel-ZZ</option>
-            <option value="46">Protein temperature</option>
-            <option value="47">Non-protein temperature</option>
-            <option value="48">Protein lambda values</option>
-            <option value="49">Non-protein lambda values</option>
+	        <option value="Bond">Bond</option>
+            <option value="Angle">Angle</option>
+            <option value="Proper-Dih.">Proper dihedrals</option>
+            <option value="Improper-Dih.">Ryckaert-Bellemans dihedrals</option>
```

Comment:
```suggestion
            <option value="Ryckaert-Bell.">Ryckaert-Bellemans dihedrals</option>
```
@thepineapplepirate?

Reply:
this could be a problem, as the name can be different depending on the version used (see above in my screen shot, the options is Improper-Dih in my case, but in yours/gmx2022, its Ryckaert-Bell), could both values be assigned? not sure how to circumvent this.

Reply:
I don't think we can assign both values. We need to identify what the difference is between these two versions. If it is some dependency, maybe we can add an additional pin in the requirements.

Can you send me the output of `conda list` for an environment on a Linux machine?